### DISCUSSION TITLE
Crashes when smash ball loading animation

### DISCUSSION THREAD

**Discussion by Edward152** [Raytwo/ARCropolis#594] [Issues]
Smash bros just crashes when i start it i just downlaoded teh lates version of acropolis my versions are firm 18.1.0 atmos ver 1.7 smash ver 13.0.3 acro ver 4.0.3

**Comment by Raytwo** [comment]
I've had a few reports but no clear idea on what the issue is yet, will take a look.

**Reply by Edward152** [reply]
I did some testing updated my atmos to 1.7.1 and removed my mods still didnt work
